src/cmd/run.rs

```diff
@@ -136,7 +136,6 @@ struct ScanGitVCSReference {
 }
 
 impl ScanGitVCSReference {
-    /// Instantiate ScanGitVCSReference instance
```

Comment:
I initially had this comment because I made this a public function because I needed it to grab the jobID and clippy/ rustfmt wanted me to provide a comment. I'm not using it anymore so I opted to just delete the comment and have it restored back to what ti was originally